### ISSUE TITLE
Editor not loaded correctly

### ISSUE THREAD

**Issue by Semir1212** [humhub/questions#11]
v 1.0.1

- Install Q&A and e.g. Polls
- Disable ability to create "Posts"
- Open Space (with a user account that can not create normal posts), click on input field for "Question" in editor without clicking anywhere else

--> Instead of the Q&A Editor the Polls editor loaded.

**Comment by yurabakhtin** [comment]
@Semir1212 I think you were confused because first input of the Polls module has similar same placehoder "Question", but the module Q&A has a placeholder "Question headline...", please compare:

The poll modules is sorted first by default:
![poll](https://github.com/humhub/questions/assets/6995524/7ade9114-7e3a-435d-b28f-b9436013f0b5)

When I sort the module Q&A before Polls:
![q a](https://github.com/humhub/questions/assets/6995524/284fc97f-276d-4de6-9245-8d468fb1c9e8)

Please note the sort order is defined here https://github.com/humhub/questions/blob/master/widgets/WallEntry.php#L38 and here for Polls - https://github.com/humhub/polls/blob/master/widgets/WallEntry.php#L32. If need we can create a module config setting to update the sort order.

**Comment by Semir1212** [comment]
@yurabakhtin The sort order of the tabs might be the problem, that the Polls editor is loaded instead of the Q&A editor. But it is not me missclicking, I just retried it with the same result. I might also just be a problem with the highlighting of the tabs. 

1. Open Space:

![image](https://github.com/humhub/questions/assets/38326726/ff57ec48-9b97-4b0e-a21c-5b290d521899)

It is already "Question" instead of "Question headline", althought the Q&A tab is preselected by the software.

2. Click on "Question":

![image](https://github.com/humhub/questions/assets/38326726/422042a6-bf15-46d4-ad8d-4d2f2fbd80a8)

I am also able to create a "Poll" although the "Q&A" Tab is selected. Note that it is necessary to directly click on "Question" after opening a space. If you click on one of the other tabs before, it works as intended.

**Comment by yurabakhtin** [comment]
@Semir1212 Ok, thank you for the additional info, yes it is strange why the sort function works differently in the 2 places(menu items and forms).

It seems the problem because the modules Polls and Q&A have the same sort order `200`. We could change one of them to another value, but I have decided to fix this in core to avoid similar issues in future with other modules.

@luke- Core PR https://github.com/humhub/humhub/pull/6574 - there I have improved the function `Sort::sort()` to able sort with several columns, for example, in current case it sort by `[200, 'Poll']` and `[200, 'Q&A']` instead of previous when we sorted only by single value `200`.
